### DISCUSSION TITLE
Controller-Version in config and updates

### DISCUSSION THREAD

**Discussion by LuDomMedia** [Art-of-WiFi/UniFi-API-client#111] [Q&A] [answered]
Hey, it's me for the third time,

I would like to know what the controller version in the config is for and what happens if it is not adjusted after updating the unifi controller.

And if there are some problems with not adjusting the config, is there any possibility to change it automatically?

**Comment by malle-pietje** [comment, marked as answer]
Currently, that value isn't being used. It was there when I "adopted" the library and decided to leave it there for compatibility's sake. We might use it in the future to throw an exception when the code uses a method that isn't supported for the current controller version.

**Reply by LuDomMedia** [reply]
Okay, thank you very much for your answer!
